README.md

```diff
@@ -1,72 +1,269 @@
+- [tree-sitter-hoon](https://github.com/urbit-pilled/tree-sitter-hoon): Tree-sitter grammar for Hoon
+- [Urbit FAQ](https://github.com/ashelkovnykov/urbit-faq): A trove of obscure Urbit information that's still working its way into the docs
+- [Urbit Wordlists](https://github.com/ashelkovnykov/urbit-wordlists): Wordlists for Urbit phonemes, plus wordlists for planets with interesting sigils
+- [Venetia](https://github.com/tylershuster/venetia): A simple command-line utility for generating a list of planet names issuable from an Urbit star.
+    - [Gonetia](https://github.com/tomholford/gonetia): Another utility to do the same: inspired by Venetia, but written in Go instead of JS
 
-## Tools
+### Nouns
 
-- [Sigil Generator](https://sigil.azimuth.network/) — a tool for generating PNG or SVG sigils in different colors and sizes.
-- [Venetia](https://github.com/tylershuster/venetia) — a simple command-line utility for generating a list of planet names issuable from an Urbit star.
-- [Gonetia](https://github.com/tomholford/gonetia) — another  command-line utility for generating a list of issuable planets from a star. Inspired by [Venetia](https://github.com/tylershuster/venetia) (but implemented in Go instead of JS).
-- [Urbit Content Archiver](https://github.com/robkorn/urbit-content-archiver) — CLI application that exports channels from your Urbit ship and auto-downloads any directly linked content.
-- [Urbit Webhook Funnel](https://github.com/robkorn/urbit-webhook-funnel) - CLI application which funnels external webhook event data to an Urbit chat.
-- [graph-store query](https://github.com/h5gq3/graph-query) - shoe/CLI app and generator that filters and queries graph-store nodes
-- [ASCII Sigil Generator](https://github.com/textprotocol/sigil) - a tool for generating ASCII sigils in the command line.
-- [UrSR](https://github.com/hosted-fornet/ursr) - Urbit Speech Recognition
-- [Urbit Star Feed](https://twitter.com/UrbitStarFeed) - Twitter feed of Urbit stars sold on OpenSea, updated every five minutes.
+- [Pynoun](https://github.com/urbit/tools/blob/master/pkg/pynoun/noun.py): Python library for natively managing nouns
 
-## Development
+### SAIL
 
-- [urbit-static-page](https://github.com/matildepark/urbit-static-page/) – Mount an arbitrary HTML page to an Urbit ship
-- [create-landscape-app](https://github.com/urbit/create-landscape-app) – Get started making a simple app for Landscape with a few commands
-- [urbit-chatbot-framework](https://github.com/robkorn/urbit-chatbot-framework) – A framework that allows anyone to create an Urbit Chatbot with only a few lines of code
+- [gin & tonic](https://github.com/arthyn/gin-tonic): 2 utilities for making SAIL development easier: auto-sync and auto-refresh
+- [Mast](https://github.com/R-JG/mast): A library for building fully dynamic SAIL frontends
+- [Rudder](https://github.com/Fang-/suite): A framework for SAIL development (Hoon-only HTML)
 
-### Integrations
+## Educational Resources
```

Comment:
Seems like an oversight not to include Hoon Academy and App Academy from urbit.org here.

Reply:
I was operating under the impression that this page is for stuff not in the official docs. Do other "awesome" projects usually link to stuff that would be equivalent to Hoon / App Academy? I thought it was more like, "look what people have done on their own!"